src/gui/optionsdlg.h

```diff
@@ -60,6 +60,8 @@ namespace Ui
 class OptionsDialog: public QDialog
 {
     Q_OBJECT
+    using ThisType = OptionsDialog;
```

Comment:
Sorry, it took me some time to realise what do I feel about those two names, and while thinking I wrote comment on another subject, which mislead you. See my response to the identifier name change above.